packages/grafana-ui/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -40,9 +42,27 @@ export const MultiCombobox = <T extends string | number>(props: MultiComboboxPro
 
   const styles = useStyles2(getComboboxStyles);
 
-  const [items, _baseSetItems] = useState(isAsync ? [] : options);
+  const allOption = useMemo(() => {
+    return {
+      label: t('multicombobox.all.title', 'All'),
+      // Type casting needed to make this work when T is a number
+      value: 'all' as unknown as T,
```

Comment:
We should extract this as a constant and set it to something obscure like `__ALL`